src/main/java/edu/ucsb/mapache/controllers/SlackSlashCommandController.java

```diff
@@ -33,13 +55,21 @@
     @Autowired
     ChannelRepository channelRepository;
 
+    @Autowired
+    GoogleSearchService googleSearchService;
+
+
     /**
      * The token you get while creating a new Slash Command. You should paste the
      * token in application.properties file.
      */
```

Comment:
delete this comment..  it's false.  the token goes in secrets-localhost.properties and secrets-heroku.properties